### PR TITLE
Pcq 1645

### PR DESCRIPTION
**Pull request by priyankaVerma21**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PCQ-1645

### Change description ###
This change is to upgrade springBoot to 3.0.6

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by rapolaskaseliscgi — APPROVED** [review]

File: charts/pcq-backend/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for pcq-backend App
 name: pcq-backend
 home: https://github.com/hmcts/pcq-backend
-version: 0.1.11
+version: 0.1.12
```
Comment (MartinYSpasov) [inline charts/pcq-backend/Chart.yaml:6]: Please upgrade to major version

File: build.gradle [hunk 2]
```diff
@@ -265,62 +265,47 @@ dependencyManagement {
     // CVE-2021-29425
     dependency group: 'commons-io', name: 'commons-io', version: '2.11.0'
 
-    dependencySet(group: 'org.springframework.cloud', version: '3.1.5') {
+    dependencySet(group: 'org.springframework.cloud', version: '4.0.2') {
```
Comment (MartinYSpasov) [inline build.gradle:268]: Do we need org.springframework.cloud, this has been removed in LAU

File: build.gradle [hunk 3]
```diff
@@ -265,62 +265,47 @@ dependencyManagement {
...
     }
-
-    dependencySet(group: 'org.springframework.security', version: '6.0.3') {
-      entry 'spring-security-core'
-      entry 'spring-security-test'
-    }
   }
   imports {
-    mavenBom 'org.springframework.cloud:spring-cloud-dependencies:2021.0.3'
+    mavenBom 'org.springframework.cloud:spring-cloud-dependencies:2022.0.2'
```
Comment (MartinYSpasov) [inline build.gradle:274]: This has been removed in LAU

File: src/main/java/uk/gov/hmcts/reform/pcqbackend/security/SecurityConfiguration.java [hunk 4]
```diff
@@ -1,57 +1,46 @@
...
 import javax.servlet.http.HttpServletResponse;
+import java.security.Security;
 
 @Configuration
 @EnableWebSecurity
-@Order(1)
-public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+@SuppressWarnings({"PMD.SignatureDeclareThrowsException"})
+
+public class SecurityConfiguration {
```
Comment (MartinYSpasov) [inline src/main/java/uk/gov/hmcts/reform/pcqbackend/security/SecurityConfiguration.java:19]: Please add public **void configure(WebSecurity web)** back. As already mentioned, I'd rather leave this here for now and remove this configuration as part of a separate ticket.

File: build.gradle (hunk 2 above)
Comment (priyankaVerma21) [reply build.gradle:268]: Hi Martin , yes it is required and it's there in LAU as well.

File: build.gradle (hunk 3 above)
Comment (priyankaVerma21) [reply build.gradle:274]: Hi Martin , yes it is required and it's there in LAU as well.

File: src/main/java/uk/gov/hmcts/reform/pcqbackend/security/SecurityConfiguration.java (hunk 4 above)
Comment (priyankaVerma21) [reply src/main/java/uk/gov/hmcts/reform/pcqbackend/security/SecurityConfiguration.java:20]: Added this back

**Review by MartinYSpasov — APPROVED** [review]

**Pull request merged**